### ISSUE TITLE
New version of Webpack (3.8.0) is breaking Encore

### ISSUE THREAD

**Issue by TheMaxoor** [symfony/webpack-encore#184]
Hi guys, 

It looks like there is an issue with the just released Webpack 3.8.0. 

### ℹ️ Configuration
**OS :** MacOS High Sierra 10.13 (17A405)
**Node Version :** v8.6.0
**NPM Version :** v5.5.1
**Encore Version :** 0.16.0

### ❌ Here is the detailed error: 
```
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.stats should be one of these:
   object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? } | boolean | "none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose"
   Used by the webpack CLI program to pass stats options.
   Details:
    * configuration.stats has an unknown property 'errors'. These properties are valid:
      object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? }
    * configuration.stats has an unknown property 'warnings'. These properties are valid:
      object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? }
    * configuration.stats should be a boolean.
    * configuration.stats should be one of these:
      "none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose"
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! BW@ start: `encore dev-server --hot`
```

### ✅  Here is a workaround to avoid this error for now : 
Add this line in your dependencies in your project's `package.json` : 
`"webpack": "3.7.0"`
Then run `npm install` or `yarn install`

### 🔁 To reproduce this issue : 
1. Install/Update your dependencies
2. Add `webpack.config.js`
3. Run Encore server

I'm currently in a rush at work, I can't investigate more on this issue for now. If someone has a fix for that, feel free to contribute 😉 

Our friends at Laravel Mix have the exact same issue right now : https://github.com/JeffreyWay/laravel-mix/issues/1279

**Comment by Aerendir** [comment]
I'm confirming this: I'm receiving the exact same error.

@TheMaxoor Thank you for the workaround! It is as as manna for guys like me not so good at using Javascript!

**Comment by Lyrkan** [comment]
Thank you for reporting this @TheMaxoor

It looks like it's now fixed in webpack 3.8.1: webpack/webpack#5840
